__tests__/todo-config-test.ts

```diff
@@ -30,41 +30,135 @@ describe('todo-config', () => {
     it('returns default object when no lint todo config found', () => {
       const config = getTodoConfig(project.baseDir);
 
-      expect(config).toEqual({
+      expect(config.daysToDecay).toEqual({
         warn: 30,
         error: 60,
       });
     });
 
-    it('can returns empty lint todo config from package.json when empty config explicitly configured', () => {
-      project.writeTodoConfig({});
+    it('can return empty lint todo config from package.json when empty config explicitly configured', () => {
+      project.writePackageJsonTodoConfig({});
 
       const config = getTodoConfig(project.baseDir);
 
-      expect(config).toEqual({});
+      expect(config.daysToDecay).toEqual({});
+    });
+
+    it('can return empty lint todo config from .lint-todorc.js when empty config explicitly configured', () => {
+      project.writeLintTodorc({});
+
+      const config = getTodoConfig(project.baseDir);
+
+      expect(config.daysToDecay).toEqual({});
     });
 
     it('can get lint todo config from package.json', () => {
-      project.writeTodoConfig({
+      project.writePackageJsonTodoConfig({
         warn: 5,
         error: 10,
       });
 
       const config = getTodoConfig(project.baseDir);
 
-      expect(config).toEqual({
+      expect(config.daysToDecay).toEqual({
         warn: 5,
         error: 10,
       });
     });
 
+    it('can get lint todo config from package.json with decay days by rule', () => {
+      project.writePackageJsonTodoConfig(
+        {
+          warn: 5,
+          error: 10,
+        },
+        {
+          'no-bare-strings': {
+            warn: 10,
+            error: 20,
+          },
+        }
+      );
+
+      const config = getTodoConfig(project.baseDir);
+
+      expect(config).toMatchInlineSnapshot(`
+        Object {
+          "daysToDecay": Object {
+            "error": 10,
+            "warn": 5,
+          },
+          "daysToDecayByRule": Object {
+            "no-bare-strings": Object {
+              "error": 20,
+              "warn": 10,
+            },
+          },
+        }
+      `);
+    });
+
+    it('can get lint todo config from .lint-todorc.js', () => {
+      project.writeLintTodorc({ warn: 20, error: 40 });
+
+      const config = getTodoConfig(project.baseDir);
+
+      expect(config.daysToDecay).toEqual({ warn: 20, error: 40 });
+    });
+
+    it('can get lint todo config from .lint-todorc.js with decay days by rule', () => {
+      project.writePackageJsonTodoConfig(
```

Comment:
is this the correct function for this particular test?

Reply:
Good catch. Updated.